xla/service/gpu/gemm_rewriter.cc

```diff
@@ -608,13 +608,14 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
   Status HandleConvert(HloInstruction *instr) override {
     HloInstruction *clamp_lower, *clamp_upper, *d_scale, *existing_gemm,
         *binary;
-    auto cuda_compute_capability_ =
-        std::get<se::CudaComputeCapability>(gpu_version_);
+    bool gpu_capability_flag_ = (std::holds_alternative<se::CudaComputeCapability>(gpu_version_) &&
```

Comment:
Maybe use a better var name?

Reply:
@cheshire Thanks for the feedback! 

May I ask could you be more specific or any suggestion?  is ```gpu_cc_convert_flag_``` ok?

Reply:
supports_different_output_type?

Reply:
Done.